app/assets/stylesheets/argo/argo.sass

```diff
@@ -183,3 +185,13 @@ button.close
 #bulk-spreadsheet-warning
   color: $color-cardinal-red
 
+.bulk-buttons
+  margin-left: 5px
+  margin-right: 5px
+
+.bulk-buttons-row
+  margin-top: 16px
+  margin-bottom: 16px
+
+.bulk-btn-danger
```

Comment:
Since we are now using buttons here, we could remove this in favor of `btn-danger`